manifests/a/angeldev0/messyorganizer/1.0.0/angeldev0.messyorganizer.locale.en-US.yaml

```
@@ -0,0 +1,32 @@
+# Created using wingetcreate 1.6.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.6.0.schema.json
+
+PackageIdentifier: angeldev0.messyorganizer
+PackageVersion: 1.0.0
+PackageLocale: en-US
+Publisher: Angel C
+PublisherUrl: https://github.com/4ngel2769
+PublisherSupportUrl: https://github.com/4ngel2769

```

```suggestion
PublisherSupportUrl: https://github.com/4ngel2769/messy-file-organizer/issues
```
This is generally preferred to be the link where users can submit new support requests or file new issues